challenges/Score-P_and_Vampir_Basics/miniSweep/cmake_openmp.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -l
+#------------------------------------------------------------------------------
+
+# CLEANUP
+rm -rf CMakeCache.txt
+rm -rf CMakeFiles
+
+# SOURCE AND INSTALL
+if [ "$SOURCE" = "" ] ; then
+  SOURCE=../minisweep
+fi
+if [ "$INSTALL" = "" ] ; then
+  INSTALL=../install
+fi
+
+if [ "$BUILD" = "" ] ; then
+  BUILD=Debug
+  #BUILD=Release
+fi
+
+if [ "$NM_VALUE" = "" ] ; then
+  NM_VALUE=4
+fi
+
+if [ "$PE_ENV" = INTEL ] ; then
+  CC=icc
+  OMP_ARGS="-qopenmp"
+  OPT_ARGS="-ip -prec-div -O3 -align -ansi-alias -fargument-noalias -fno-alias -fargument-noalias"
+else
+  CC=scorep-mpicc
+  OMP_ARGS="-fopenmp"
+  OPT_ARGS="-O3 -fomit-frame-pointer -funroll-loops -finline-limit=10000000"
+fi
+
+#------------------------------------------------------------------------------
+
+SCOREP_WRAPPER=off cmake \
```

Comment:
You may want to remove the SCOREP_WRAPPER and the scorep-mpicc from this since this is an exercise.